style.css

```diff
@@ -220,34 +290,342 @@ h2 {
+  color: #172b4d;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+}
+
+textarea {
+  padding: 10px;
+  color: #172b4d;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+}
+
+.contact-btn {
+  padding: 12px;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  border-radius: 12px;
+  margin-bottom: 45px;
+  align-self: center;
+}
+
+.contact-btn:disabled {
+  align-self: flex-start;
+  padding: 12px;
+  color: #5e6c84;
+  border: solid #5e6c84 2px;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  border-radius: 12px;
+  margin-bottom: 45px;
+}
+
+.contact-btn:hover {
+  align-self: flex-start;
+  padding: 12px;
+  color: #fff;
+  border: solid #396df2 2px;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  border-radius: 12px;
+  margin-bottom: 45px;
+}
+
+.hdr-bgr {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-self: stretch;
+  background-image: url('Img/Headerbg@2x.png');
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+  padding: 40px 0 100px 0;
+}
+
+.bgr {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: stretch;
+  background-color: #6070ff;
+  border-top-left-radius: 80px;
+  gap: 15px;
+}
+
+.bgr-img {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url('Img/contactformbackgroundshapes.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+}
+
+.about-grid {
+  display: grid;
+  align-self: center;
+  background-color: #fff;
+  grid-template-columns: auto;
+  width: 80%;
+  padding: 40px 0 72px 0;
+  margin-top: 20px;
+}
+
+.about-item1 {
+  grid-template-areas: 1 / 1 / 2 / 2;
+  display: flex;
+  justify-content: center;
+}
+
+.info {
+  flex: 1;
+  padding: 0 5% 0 0;
+  margin: 0;
+}
+
+.skills {
+  flex: 1;
+  padding: 0;
+  margin: 0;
+}
+
+.skills-list {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin: 0;
+}
+
+.lng-list {
+  display: inline-flex;
+  gap: 10px;
+  padding: 0;
+  margin: 0;
+}
+
+.lng-list img {
+  padding: 10px;
+}
+
+.line {
+  display: flex;
+  align-self: stretch;
+  background-color: #dfe1e6;
+  height: 1px;
+}
+
+.skills p {
+  padding: 10px 0;
+}
+
+.enable {
+  transform: rotate(-90deg);
+}
+
+.lng-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  align-self: flex-start;
+  width: 122px;
+  height: 120px;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.lng-item p {
+  margin: 0;
+  padding: 10px 0 0 10px;
+}
+
+.lng-item img {
+  margin: 0;
+  padding: 0 0 0 10px;
+}
+
+@media only screen and (max-width: 768px) {
+  /* Welcome section */
+  .hdr-bgr {
+    background-image: none;
   }
 
```

Comment:
- [ ] [OPTIONAL] It is best to start responsive design using the mobile-first approach to make it easier.